src/main/java/racingcar/model/Car.java

```diff
@@ -0,0 +1,17 @@
+package racingcar.model;
+
+public class Car {
+    public static final int MOVE_STANDARD = 4;
+    public static final int MOVE_DISTANCE = 1;
+    private int distance = 0;
```

Comment:
```suggestion
    public static final int MOVE_STANDARD = 4;
    public static final int MOVE_DISTANCE = 1;
    
    private int distance = 0;
```

1. 개행을 둬서, 좀더 가독성을 높여보면 어떨까요?
2. 굳이 상수가 public이여야 하는 이유가 있을까요?